source-browser.html

```diff
@@ -181,7 +209,46 @@ <h1 id="loading" class="m1 center display-none">⌛</h1>
           .focus();
       }
 
+      function createNavbar(topic) {
+        $('.li').append(
+          $('<a>', {
+            id: `li${topic}`,
+            href: `#${topic}`
+          }).text(topic)
+        );
+      }
+
+      let list = [];
+      $(document).ready(() => {
+	    TOPICS.forEach(item => {
+	      let heading = item.name;
+	      list.push(heading);
+	    });
+      });
+
+      $(window).scroll(() => {
```

Comment:
Should this also happen on window resize?

Reply:
I've added that, as well.